test/starlark_tests/watchos_static_framework_tests.bzl

```diff
@@ -49,6 +49,34 @@ def watchos_static_framework_test_suite(name):
         tags = [name],
     )
 
+    # Tests Swift watchos_static_framework builds correctly for arm64, and i386 cpu's.
+    archive_contents_test(
+        name = "{}_swift_arm64_builds".format(name),
+        build_type = "simulator",
+        target_under_test = "//test/starlark_tests/targets_under_test/watchos:swift_static_fmwk",
+        cpus = {
+            "watchos_cpus": ["x86_64", "arm64"],
+        },
+        binary_test_file = "$BUNDLE_ROOT/swift_static_fmwk",
+        binary_test_architecture = "arm64",
+        macho_load_commands_contain = ["cmd LC_BUILD_VERSION", "platform WATCHOSSIMULATOR"],
+        macho_load_commands_not_contain = ["cmd LC_VERSION_MIN_WATCHOS"],
```

Comment:
is there a `platform` line that should be here too? I feel like I saw somewhere that all platforms should be using `LC_BUILD_VERSION` moving forward in which case this one wouldn't be here ever, even if this test was incorrect

Reply:
I copied the tvOS and iOS tests. I think they are just confirming the difference between arm64 and non-arm64. If we want to change these, we should probably change all of them?

Reply:
I would just be fumbling, since I don't know what these are. Can you/someone make a PR to change them all together after this one panda?